src/pretix/control/templates/pretixcontrol/item/index.html

```diff
@@ -1,146 +1,169 @@
-                    </div>
-                    {% include "pretixcontrol/includes/logs.html" with obj=item %}
+                <div class="col-md-8">
+                  {% bootstrap_form form layout="inline" %}
                 </div>
+              </div>
+              {% endfor %}
             </div>
+          </div>
+          {% endif %}
+        </fieldset>
+        <fieldset>
+          <legend>{% trans "Price" %}</legend>
+          {% bootstrap_field form.default_price
+          addon_after=request.event.currency layout="control" %} {%
+          bootstrap_field form.tax_rule layout="control" %} {% bootstrap_field
+          form.free_price layout="control" %} {% bootstrap_field
+          form.original_price addon_after=request.event.currency
+          layout="control" %}
+        </fieldset>
+        <fieldset>
+          <legend>{% trans "Availability" %}</legend>
+          {% bootstrap_field form.sales_channels layout="control" %} {%
+          bootstrap_field form.available_from layout="control" %} {%
+          bootstrap_field form.available_until layout="control" %} {%
+          bootstrap_field form.max_per_order layout="control" %} {%
+          bootstrap_field form.min_per_order layout="control" %} {%
+          bootstrap_field form.require_voucher layout="control" %} {%
+          bootstrap_field form.hide_without_voucher layout="control" %} {%
+          bootstrap_field form.require_bundling layout="control" %} {%
+          bootstrap_field form.allow_cancel layout="control" %} {%
+          bootstrap_field form.allow_waitinglist layout="control" %} {%
+          bootstrap_field form.hidden_if_available layout="control" %}
+        </fieldset>
+        {% for v in formsets.values %}
+        <fieldset>
+          <legend>{{ v.title }}</legend>
+          {% include v.template with formset=v %}
+        </fieldset>
+        {% endfor %}
+        <fieldset>
+          <legend>{% trans "Tickets & check-in" %}</legend>
+          {% bootstrap_field form.generate_tickets layout="control" %} {%
+          bootstrap_field form.checkin_attention layout="control" %}
+        </fieldset>
+        <fieldset>
+          <legend>{% trans "Additional settings" %}</legend>
+          {% bootstrap_field form.issue_giftcard layout="control" %} {%
+          bootstrap_field form.show_quota_left layout="control" %} {% for f in
+          plugin_forms %} {% bootstrap_form f layout="control" %} {% endfor %}
+        </fieldset>
+      </div>
+      <div class="form-group submit-group">
+        <button type="submit" class="btn btn-primary btn-save">
+          {% trans "Save" %}
+        </button>
+      </div>
+    </div>
+    <div class="col-xs-12 col-lg-2">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">{% trans "Product history" %}</h3>
         </div>
-    </form>
+        {% include "pretixcontrol/includes/logs.html" with obj=item %}
+      </div>
```

Comment:
All of the changes in this file should be reverted, I don't think there is anything related to eventyay branding included here.